src/explore-education-statistics-common/src/components/Tabs.tsx

```diff
@@ -14,9 +14,10 @@ import TabsSection, { TabsSectionProps } from './TabsSection';
 interface Props {
   children: ReactNode;
   id: string;
+  onToggle?: any;
```

Comment:
`any` should throw up a lint warning and is consequently highly discouraged unless there's a valid reason to do so. Use the correct type here